package.json

```diff
@@ -28,17 +28,22 @@
   },
   "dependencies": {
     "@hookform/resolvers": "^3.6.0",
+    "@nextui-org/pagination": "^2.0.33",
     "@stanfordbdhg/design-system": "file:./packages/design-system",
     "@t3-oss/env-nextjs": "^0.10.1",
+    "@tanstack/match-sorter-utils": "^8.15.1",
     "@tanstack/react-table": "^8.19.2",
     "firebase": "^10.12.2",
+    "firebase-admin": "^12.2.0",
```

Comment:
TODO to ensure that we remove the admin SDK before we merge the PR; we should ensure that we don't use the admin SDK in the client-facing web dashboard.

Reply:
`firebase-admin` is used for retrieving user's auth data. We can't get it through Firestore. For example, to retrieve patients data, we need to combine data from user's auth and `user` collection. 

User can access other users data only if authenticated, for their respective roles and `firebase-admin` is executed on the server only. I agree it probably would be better if it was separate Firebase function, but that shouldn't be blocking for now.

Reply:
@pauljohanneskraft Would it make sense & doable to extract that functionality in a cloud function so we can avoid that work in the frontend codebase & ensure that we don't need to embed admin SDK keys?

On a general level: I think we should generally use the firebase queries using the client SDK when possible to ensure that we build on top of the Security Rules but in some edge cases we should selectively use cloud functions if needed.